packages/i18nify-js/README.md

```diff
@@ -541,6 +543,47 @@ console.log(getDialCodeByCountryCode('BR')); // Outputs the dial code for Brazil
 console.log(getDialCodeByCountryCode('DE')); // Outputs the dial code for Germany (+49)
 ```
 
+#### getMaskedPhoneNumber(options)
+
+📞🔒 The getMaskedPhoneNumber function is a versatile tool designed to handle phone number formatting and masking based on the specific requirements of different countries. This function is ideal for applications that require the display of partially hidden phone numbers for security purposes or privacy concerns. It supports a wide range of configurations, including options to mask portions of the phone number, specify the number of digits to mask, and choose whether to mask digits from the beginning or end of the number.
+
+##### Examples
+
+```javascript
+// Masking a U.S. phone number completely
+console.log(
+  getMaskedPhoneNumber({
+    countryCode: 'US',
+    phoneNumber: '2025550125',
+    withDialCode: true,
+  }),
+);
+// Output: +1 xxx-xxx-xxxx
+
+// Partially masking an Indian phone number, hiding the last 6 digits
+console.log(
+  getMaskedPhoneNumber({
+    countryCode: 'IN',
+    phoneNumber: '9876543210',
+    maskingOptions: {
+      maskingStyle: 'suffix',
+      maskedDigitsCount: 6,
+      maskingChar: '*',
+    },
+    withDialCode: true,
+  }),
+);
+// Output: +91 9876**-****
```

Comment:
Why are we getting `-` in the output ? 
We don't get this `-` in formatted output right ?

Reply:
We do have `-` in formatting. See below screenshot.

<img width="194" alt="Screenshot 2024-05-14 at 9 59 04 AM" src="https://github.com/razorpay/i18nify/assets/61814592/6f9bfa70-ed8d-493c-b1a0-5a3e2f8b82d7">